march:
  hardware_interface:
    loop_hz: 100 # hz
    joints:
      - test_joint
